app/Http/Controllers/Admin/UsersController.php

```diff
@@ -20,9 +20,6 @@ class UsersController extends Controller
     public function __construct()
```

Comment:
Constructor eruit en afhandelen met een middleware in de routes (Gebeurt waarschijnlijk al)